ql/src/java/org/apache/hadoop/hive/ql/parse/HiveLexer.g

```diff
@@ -333,6 +333,7 @@ KW_VALIDATE: 'VALIDATE';
 KW_NOVALIDATE: 'NOVALIDATE';
 KW_RELY: 'RELY';
 KW_NORELY: 'NORELY';
+KW_UNIQUE: 'UNIQUE';
```

Comment:
On jira, there is a flag for Incompatible changes and a field for release notes. At the time of commit, update those fields.